src/reducers/index.js

```diff
@@ -29,7 +29,6 @@ import { warningsByUserIdReducer } from './warningsReducer';
 import { infoCollectionsReducer } from './informationReducer';
 import { weeklySummariesAIPromptReducer } from './weeklySummariesAIPromptReducer';
 import { mouseoverTextReducer } from './mouseoverTextReducer';
-import notificationReducer from './notificationReducer';
```

Comment:
Hi Llya,
I wanted to seek some information on the purpose of notificationReducer in index.js and it's removal from the file? It would be helpful to me, if you could share your understanding on the purpose of the reducer.

Reply:
@Parth-tech Hi! Thank you for bringing this to my attention. Honestly, I didn't touch /reducers/index.js, this change must have occurred accidentally when I was resolving merge conflicts. I have restored it back to what it is on the development branch right now